console/src/main/webapp/manager/app/components/area/area.es6

```diff
@@ -89,6 +85,17 @@ class AreaController {
         f.setId(key)
         if (this.ids.indexOf(f.getId()) >= 0) selected.push(f)
       })
+
+      if (!config.map) {
+        this.map.getView().fit(vector.getSource().getExtent(), map.getSize())
+      } else {
+        let view = new ol.View({
+          center: ol.proj.fromLonLat(config.map.center),
+          zoom: config.map.zoom
+        })
+        this.map.setView(view)
```

Comment:
You can just update view center & zoom rather than creating a new one.

Reply:
Done with : ec79291